src/locale/locales/zh-CN/messages.po

```diff
@@ -0,0 +1,4230 @@
+msgid ""
+msgstr ""
+"Project-Id-Version: \n"
+"Report-Msgid-Bugs-To: \n"
+"POT-Creation-Date: 2024-01-29 23:06+0800\n"
+"PO-Revision-Date: \n"
+"Last-Translator: Frudrax Cheng <i@cynosura.one>\n"
+"Language-Team: Frudrax Cheng, Simon Chan\n"
+"Language: zh_CN\n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"Plural-Forms: nplurals=1; plural=0;\n"
+"X-Generator: Poedit 3.4.2\n"
+
+#: src/view/com/modals/VerifyEmail.tsx:142
+msgid "(no email)"
+msgstr "(没有邮件)"
+
+#: src/view/shell/desktop/RightNav.tsx:168
+msgid "{0, plural, one {# invite code available} other {# invite codes available}}"
+msgstr "{0, plural, one {# 邀请码可用} other {# 邀请码可用}}"
+
+#: src/view/com/profile/ProfileHeader.tsx:632
+msgid "{following} following"
+msgstr "{following} 正在关注"
+
+#: src/view/shell/desktop/RightNav.tsx:151
+msgid "{invitesAvailable, plural, one {Invite codes: # available} other {Invite codes: # available}}"
+msgstr "{invitesAvailable, plural, one {邀请码: # 可用} other {邀请码: # 可用}}"
+
+#: src/view/screens/Settings.tsx:435 src/view/shell/Drawer.tsx:664
```

Comment:
One other thing I noticed that you might want to check: all the other messages.po files seem to have each code reference on a separate line, and I'm not sure but I think that might be the format expected by Lingui? So it's possible that you might need to add some line breaks, like here, for instance

Reply:
I referenced the format of other "messages.po" files and added extra line breaks between strings. sorry for any oversight on my part.

Reply:
No need to apologise, it's just that I had noticed other contributors who had used Poedit had run into that issue before with the line breaks, so I wanted to mention it, that's all :)